src/ducks/selection/SelectionBar.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useContext, useCallback } from 'react'
+
+import UISelectionBar from 'cozy-ui/transpiled/react/SelectionBar'
+import Alerter from 'cozy-ui/transpiled/react/Alerter'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+
+import { SelectionContext } from 'ducks/context/SelectionContext'
+import useSelectionBarActions from 'ducks/selection/useSelectionBarActions'
+import { useTransactionCategoryModal } from 'ducks/transactions/TransactionRow'
+
+const SelectionBar = () => {
+  const {
+    isSelectionModeEnabled,
+    isSelectionModeActive,
+    selected,
+    emptySelection
+  } = useContext(SelectionContext)
+
+  const { t } = useI18n()
+
+  const beforeUpdate = useCallback(() => {
+    emptySelection()
+  }, [emptySelection])
+
+  const afterUpdates = () => {
```

Comment:
I think we can keep it in the singular :)

Reply:
`afterUpdate` et `beforeUpdate` s’exécutent après chaque update, alors que `afterUpdates` ne s’exécutent qu'une seule fois, après tous les updates.